lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -649,7 +649,8 @@ SPIRVType *LLVMToSPIRVBase::transSPIRVJointMatrixINTELType(
 
   auto ParseInteger = [this](StringRef Postfix) -> ConstantInt * {
     unsigned long long N = 0;
-    consumeUnsignedInteger(Postfix, 10, N);
+    if (consumeUnsignedInteger(Postfix, 10, N))
+      assert(0 && "Error in extracting integer value");
```

Comment:
Should probably be a SPIR-V module error in this case to see such scenario in release build as well (and I believe we can't put a contract here that it would never happen).